### PR TITLE
add contract library auto-generated docs

### PR DESCRIPTION
**Pull request by CJ42**
# What does this PR introduce?

Add auto-generated docs for contract libraries.

image

### CODE REVIEW

File: docs/contracts/libraries/LSP0Utils.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# LSP0Utils
+
+:::info Solidity contract
+
+[`LSP0Utils.sol`](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/undefined)
```
Comment (Hugoo) [inline docs/contracts/libraries/LSP0Utils.md:5]: There is undefined here - expected?

File: docs/contracts/libraries/LSP0Utils.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+
+### getLSP1DelegateValue
+
+
+
+
+
+
+
```
Comment (Hugoo) [inline docs/contracts/libraries/LSP0Utils.md:28]: Why so many blank lines?

File: docs/contracts/libraries/LSP17Utils.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+
+### isExtension
+
+
+
+
+
+
+
```
Comment (Hugoo) [inline docs/contracts/libraries/LSP17Utils.md:28]: Same

**Review by Hugoo — APPROVED** [review]
nice to prettify!

**Review by skimaharvey — APPROVED** [review]

**Pull request merged**